.gitignore

```diff
@@ -40,7 +40,6 @@ coverage.xml
 
 # Translations
 *.mo
-*.pot
```

Comment:
pot are not needed anymore in OCA because are automatically generated (and thus included ones are ignored) and it's preferred to be excluded